### DISCUSSION TITLE
手順 8-3.トラックボールケース装着のねじ絞めについて

### DISCUSSION THREAD

**Discussion by umi8** [Yowkees/keyball#715] [Q&A]
keyball39を組み立てています。
手順8-3.トラックボールケース装着において、【頭が平らなねじ】２本を本体裏から絞めようとしたのですが、ボール保持ケースの穴に合わず、全くねじが入っていきません。
かなり力をいれるとねじ込める感触はあるのですが、ケースが壊れそうで心配です。他の箇所のねじ締めではほとんど力を必要としなかったのですが、ボール保持ケースは力を込めながらねじを絞めたほうがいいのでしょうか？
![PXL_20241220_131606727](https://github.com/user-attachments/assets/b2ed6ae1-e2fc-4168-b51f-6e25e4944fda)
![PXL_20241220_131613131](https://github.com/user-attachments/assets/e540ed5f-0300-4c30-a9dd-e532b46f2a1c)
![PXL_20241220_131629661](https://github.com/user-attachments/assets/7c1b25b0-9da8-4ae9-89d0-f4771a912985)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

M1.7平頭ネジは「タップネジ」になりますので、ボールケースの穴にネジ切りしながら入り込みますので、多少抵抗があるかと思います。
ドライバーは0番のものを使用しないと、ネジ穴を潰しやすいです。

もし新しいネジをご希望でしたら１本単位で販売できますので、下記よりお問い合わせいただければ幸いです。
https://shirogane-lab.net/inquiry

以上、よろしくお願いいたします。

**Reply by umi8** [reply]
ご回答頂きありがとうございます。ねじの先端が磁石になっているものに変えたところ、ねじ先が安定し、締めることができました。
お手数をおかけしました。
